packages/sui-mono/bin/sui-mono-check.js

```diff
@@ -1,6 +1,26 @@
 /* eslint no-console:0 */
+const program = require('commander')
 const checker = require('../src/check')
 
+program
+  .on('--help', () => {
+    console.log('  Description:')
+    console.log('')
+    console.log('    Reports if any package should be released based on ComVer')
+    console.log('')
+    console.log('    It will create a MAJOR only if there are BREAKING CHANGES in the commit description')
```

Comment:
What do you mean with this? Do we have to write 'BREAKING CHANGES' in the commit description?